<launch>

  <node pkg="giskardpy" type="joint_goal_splitter.py" name="joint_goal_splitter" output="screen">
    <rosparam param="state_topics">["/whole_body_controller/base/state", "whole_body_controller/state"]</rosparam>
    <rosparam param="client_topics">["/whole_body_controller/follow_joint_trajectory/base", "whole_body_controller/follow_joint_trajectory"]</rosparam>
  </node>

</launch>
